cli/test/test_runner_kubernetes.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import utils as testutils
+
+from popper.config import PopperConfig
+from popper.parser import YMLWorkflow
+from popper.runner import WorkflowRunner
+
+from popper.cli import log as log
+
+
+class TestKubernetesDockerRunner(unittest.TestCase):
```

Comment:
there is a `PopperTest` that we can derive from that has the `mk_repo` function as a member. Take a look at how this is done [here](https://github.com/systemslab/popper/blob/master/cli/test/test_runner_host.py#L19).

Reply:
Done